sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp

```diff
@@ -1128,6 +1145,10 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
          * Time handle was last connected to (UTC).
          */
         DateTime LastReconnectedOn;
+        /**
+         * Array of AccessRight.
+         */
+        std::vector<AccessRight> AccessRightList;
```

Comment:
Is it a vector of access rights? Or is it a set? Can you please document what the semantics of having two AccesRight::Write elements in the vector mean?

Reply:
This is machine generated code in `_detail` namespace and invisible to customers. In convenience layer, it will be a set.